dev/common.py

```diff
@@ -52,6 +54,12 @@
         "Title": "Title",
         "Assessment Status": "Type",
     },
+    "cis_azure": {
+        "Section #": "Section",
+        "Recommendation #": "Rule Number",
+        "Title": "Title",
+        "Assessment Status": "Type",
+    },
```

Comment:
It seems a bit repetitive, maybe we can set it as the default benchmark mapping, assuming that other benchmarks in the future will use the same mapping.